contracts/claims.sol

```diff
@@ -109,22 +114,22 @@ contract Claims {
      *
      * Usage Pattern: make_claim(digest_claim(operator, tag, keccak256(abi.encodePacked(x, y, z)))).
      */
-    function make_claim(bytes32 _claim) internal returns(uint64) {
-        if (claim_status[_claim]==0) { // The claim must not have been made before
-          claim_status[_claim] = get_block_number() + challenge_period_in_blocks; // Register the claim
-          return 1;
-        }
-        else {
-          return 0;
-        }
+    function make_claim(bytes32 _claim) internal {
+        require(claim_status[_claim]==0); // The claim must not have been made before
+        claim_status[_claim] = get_block_number() + challenge_period_in_blocks;
     }
 
+
+
     /** Reject a pending claim as invalid. */
-    /* TODO: Actually implement the functionality for the rejection of claims.
     function reject_claim(bytes32 _claim) internal {
-        require_claim_pending(_claim);
+        /* require_claim_pending(_claim); */
         claim_status[_claim] = REJECTED;
-    }*/
+    }
+
+    function is_claim_rejectable(bytes32 _claim) internal view returns(bool) {
+      return claim_status[_claim] != REJECTED && claim_status[_claim] != ACCEPTABLE;
```

Comment:
NB2: *If* a claim has challenges in more than 1 step, then it also needs an optional commitment to a set of active challenges, themselves claims.

Reply:
The ACCEPTABLE entry exist since the beginning of the code. It was already there in 2018. Yes the claims are in 32 bytes and I did not change that.
I do not understand the NB2

Reply:
Of course, that might have been a premature optimization. Using a separate byte (or bool) to denote the "status" of rejected vs not-rejected-so-far may or may not be preferrable.

Reply:
It was a mistake by me to write ** && claim_status[_claim] != ACCEPTABLE** because indeed they are never set to acceptable. Therefore, I removed that test.